ex1.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="ex1_header_css.css">
+	<link rel="stylesheet" type="text/css" href="ex2_bod_css.css">
+	<link rel="stylesheet" type="text/css" href="buttons.css">
+</head>
+<body style="margin: 0">
+
+	<section>
+
+		<div class="header">
+			<div class="logindiv">
+				<div class="logintitle">
+					<h1>Interview Feedback</h1>
+				</div>
+				
+				<form action="/action_page.php" class="loginform">
+					<input type="text" placeholder="Username" name="name_box" class="loginboxes">
+					<br><br>
+					<input type="password" placeholder="Password" name="pass_box" class="loginboxes">
+					<br><br><br><br><br><br><br>
+					<a href="ex2.html" class="loginbutton">Login</a>
+				</form> 
+			</div>
+		</div>
+	</section>
+
+	<section>
+		<div class="div1">
+			<p><img src="new-feedback.jpg" style="float:left" class="image">
+				<article>
+					<br>
```

Comment:
All `<br>` tags throughout this code should go away and any spacing requirements should be moved to the CSS, where they belong, yes? :wink: